sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -6,7 +6,6 @@
 
 import { Span as OpenCensusSpan } from '@opencensus/web-types';
```

Comment:
I would expect these to be removed as well?

Reply:
Yes, I removed them and didn't regen - thanks!